.travis.yml

```diff
@@ -3,9 +3,11 @@ sudo: false
 language: go
 
 go:
-  - "1.8"
-  - "1.9"
   - "1.10"
+  - "1.11"
+  - "1.12"
+  - "1.13"
+  - "1.14"
```

Comment:
I noticed the versions of Go being tested were past their EOL so I updated this list in a separate commit. I would be happy to revert this change if it is undesirable.

Reply:
It looks like CI is failing on 13 and 14 while trying to update dependencies, so I'll need to take another look.